src/services/helper/entity.js

```diff
@@ -1,42 +1,46 @@
 // Dependencies
-const ObjectId = require('mongoose').Types.ObjectId
-
 const httpStatusCode = require('@generics/http-status')
 const common = require('@constants/common')
-const entitiesData = require('@db/entities/query')
-const KafkaProducer = require('@generics/kafka-communication')
-const utils = require('@generics/utils')
+
+const entityTypeQueries = require('../../database/queries/entity')
+const { UniqueConstraintError, ForeignKeyConstraintError } = require('sequelize')
+const { Op } = require('sequelize')
+
 module.exports = class EntityHelper {
 	/**
 	 * Create entity.
 	 * @method
 	 * @name create
 	 * @param {Object} bodyData - entity body data.
-	 * @param {String} _id - entity id.
+	 * @param {String} id -  id.
 	 * @returns {JSON} - Entity created response.
 	 */
 
-	static async create(bodyData, _id) {
-		bodyData.createdBy = ObjectId(_id)
-		bodyData.updatedBy = ObjectId(_id)
+	static async create(bodyData, id) {
+		bodyData.created_by = '0' || id
```

Comment:
[DISCUSS]

Reply:
The temporary solution has been removed.